src/components/paragraph/index.jsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
don't think we technically need this with react 17 but no big deal https://stackoverflow.com/a/64646593/7491536

Reply:
ESlint throws an error if it's not there